### PR TITLE
Calender: Add ability to set the time of day for an event

### PR DESCRIPTION
**Pull request by atdykema**
When adding an event to Calender application, the only configurable elements of the event is the name of the event and the day of the event. This pull request adds the ability to specify the time during the day the event takes place add for added specificity and customization of the "add-event" function
Screen Shot 2022-06-11 at 9 16 53 AM

### CODE REVIEW

**Review by MacDue — COMMENTED** [review]
You might want to come up with a clearer commit message. 

> Calender: Add eventtime func. to add event 

Is not very clear to me. Perhaps reword it to something like:

> Calender: Add ability to set the time of day for an event

(If that's what this change is doing)

**Comment by MacDue** [comment]
Also, with GUI changes I think it's nice add a screenshot to the PR.

**Comment by atdykema** [comment]
> Also, with GUI changes I think it's nice to add a screenshot to the PR.

Thanks for the feedback!

**Comment by MacDue** [comment]
Btw, you've only changed the title of your PR, not your commit message. To change you commit message do `git commit --amend`, change the title, then `git push -f`.

**Review by kleinesfilmroellchen — APPROVED** [review]

**Review by bgianfo — APPROVED** [review]
LGTM!
